### PR TITLE
In Browse section, the name “ Whole folder” Is not clearly visible since it is in white font color

### LINKED ISSUE #595: In Browse section, the name “ Whole folder” Is not clearly visible since it is in white font color.(refer attached pic).
The name “ Whole folder” Is not clearly visible since it is in white font color.(refer attached pic) Could we use a different mechanism to highlight the selection?
fossy

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
Added black color with bold font if selected.
background color of toggle checkbox is maintained same if selected or not
closes #595

### CODE REVIEW

File: src/www/ui/css/jquery-ui.css [hunk 1]
```diff
@@ -903,7 +903,7 @@ body .ui-tooltip {
 .ui-widget-content .ui-state-active,
 .ui-widget-header .ui-state-active {
 	border: 1px solid #aaaaaa;
-	background: #ffffff url("images/ui-bg_glass_65_ffffff_1x400.png") 50% 50% repeat-x;
+/*	background: #ffffff url("images/ui-bg_glass_65_ffffff_1x400.png") 50% 50% repeat-x;*/
```
Comment (maximilianhuber) [inline src/www/ui/css/jquery-ui.css]: this line could be removed

**Comment by shaheemazmalmmd** [comment]
Done

**Comment by maximilianhuber** [comment]
Thank you, I am currently building it for testing

**Comment by maximilianhuber** [comment]
What are marked uploads?

**Comment by shaheemazmalmmd** [comment]
In browse Menu. Every file you select using check-box  are marked uploads.

**Comment by maximilianhuber** [comment]
This feature does not work for me. Pressing the button does not change anything

**Pull request merged**